app/controllers/candidate_interface/candidate_interface_controller.rb

```diff
@@ -10,11 +10,7 @@ class CandidateInterfaceController < ActionController::Base
   private
 
     def redirect_to_dashboard_if_not_amendable
-      if FeatureFlag.active?('edit_application')
-        redirect_to candidate_interface_application_complete_path unless current_application.amendable?
-      elsif current_application.submitted?
-        redirect_to candidate_interface_application_complete_path
-      end
+      redirect_to candidate_interface_application_complete_path if current_application.submitted? && !current_application.amendable?
```

Comment:
This doesn't have the feature flag anymore, are we sure the logic is still the same?

Reply:
Yes! This is `#amendable?`:

```ruby
  def amendable?
    FeatureFlag.active?('edit_application') && submitted? && within_amend_period?
  end
```

The feature flag was redundant.